pay-api/migrations/versions/2024_03_06_04b8a7bed74e_ejv_link_generic.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: 04b8a7bed74e
+Revises: bacb2b859d78
+Create Date: 2024-03-06 11:40:00.153387
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '04b8a7bed74e'
+down_revision = 'bacb2b859d78'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('ejv_invoice_links', sa.Column('link_type', sa.String(length=20), nullable=False))
```

Comment:
is there a default value if nullable is false?

Reply:
good catch, I think I'd update it to nullable=True

Reply:
Probably also an update statement to set it all to to the type of invoice as there is no data for partial refunds yet.

Reply:
done